<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IzvestajForm</class>
 <widget class="QDialog" name="IzvestajForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="rb_mesecni_izvestaj">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>330</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Mesecni izvestaj</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>430</y>
     <width>176</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_godisnji_izvestaj">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>380</y>
     <width>139</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Godisnji izvestaj</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>4</y>
     <width>1101</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
